LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/JsonHandlers/LnsStationConfiguration.cs

```diff
@@ -183,6 +192,8 @@ private static Bandwidth GetBandwidth(uint bandwidth)
             return CastToEnumIfDefined<Bandwidth>((int)bandwidthInKHz);
         }
 
+        public static Region GetRegion(string jsonInput) => RegionConfigurationConverter.Read(jsonInput);
```

Comment:
[super-nit] consider moving this reader to the `BasicsStationConfigurationService`. That would make it very clear that the `Region` reader does not hook into an intermediate parsing of the configuration, but that it reparses the JSON string that is the LBS configuration.

Reply:
As discussed, will leave it as it is for now.